iwyu_output.cc

```diff
@@ -537,6 +538,19 @@ void IwyuFileInfo::AddForwardDeclare(const clang::NamedDecl* fwd_decl,
            << internal::GetQualifiedNameAsString(fwd_decl) << "\n";
 }
 
+void IwyuFileInfo::AddUsingDecl(const clang::UsingDecl* using_decl) {
```

Comment:
There are a few other places in this file that are fully qualifying the names.  I have made a not to remove them in a future change.